### ISSUE TITLE
VSCodium(32-bit) Shortcut not launching on Windows x64 and Windows x86 machine

### ISSUE THREAD

**Issue by metaad** [VSCodium/vscodium#1614]
I have installed VSCodium 1.81.1.23222 (x86) downloaded from (https://github.com/VSCodium/vscodium/releases/download/1.81.1.23222/VSCodium-ia32-1.81.1.23222.msi) on Windows 10 x64 VM and Windows 10 x86 VM. When I try to launch shortcut VSCodium.lnk, no GUI is coming up. In task manager VSCodium.exe is running but it is not showing up. The same behaviour has been noticed in 32bit application of 1.80.2.23209. In it's earlier version "1.77.3.23102" shortcut is launching.

I would like to know what has been changed since 1.80.2.23209 to 1.81.1.23222 in which shortcut is not launching. Can you suggest what is going missing during installation?

**Comment by xparq** [comment]
FWIW, this is what I see in the console after unpacking `VSCodium-win32-ia32-1.81.1.23222.zip` (what VSCodium has downloaded) over the prev. files, and trying to launch it:

~~~~
Loading "@vscode/policy-watcher" failed
Error: Could not locate the bindings file. Tried:
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\build\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\build\Debug\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\build\Release\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\out\Debug\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\Debug\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\out\Release\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\Release\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\build\default\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\compiled\16.17.1\win32\ia32\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\addon-build\release\install-root\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\addon-build\debug\install-root\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\addon-build\default\install-root\vscode-policy-watcher.node
 → C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\lib\binding\node-v110-win32-ia32\vscode-policy-watcher.node
    at bindings (C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\bindings\bindings.js:126:9)
    at Object.<anonymous> (C:\Program Files (x86)\VSCodium\resources\app\node_modules.asar\@vscode\policy-watcher\index.js:6:44)
    at l._compile (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:5:1257)
    at Module._extensions..js (node:internal/modules/cjs/loader:1196:10)
    at Module.load (node:internal/modules/cjs/loader:1011:32)
    at Module._load (node:internal/modules/cjs/loader:846:12)
    at f._load (node:electron/js2c/asar_bundle:2:13330)
    at Module.require (node:internal/modules/cjs/loader:1035:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at r (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:6:98)
    at n.load (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:5:1637)
    at f.load (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:4:13565)
    at l (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:6:9301)
    at Object.errorback (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:6:9420)
    at f.triggerErrorback (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:4:13859)
    at C:\Program Files (x86)\VSCodium\resources\app\out\main.js:4:13610
    at n.load (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:5:1654)
    at f.load (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:4:13565)
    at l (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:6:9301)
    at h._loadModule (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:6:9429)
    at h._resolve (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:7:452)
    at h.defineModule (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:6:5524)
    at _ (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:7:1741)
    at w (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:5:2448)
    at Object.<anonymous> (C:\Program Files (x86)\VSCodium\resources\app\out\vs\code\electron-main\main.js:42:70980)
    at Object.<anonymous> (C:\Program Files (x86)\VSCodium\resources\app\out\vs\code\electron-main\main.js:92:1872)
    at n._createAndEvalScript (C:\Program Files (x86)\VSCodium\resources\app\out\main.js:5:2492)
    at C:\Program Files (x86)\VSCodium\resources\app\out\main.js:5:2133                                                        32-ia32\\vscode-policy-watcher.node',
  ],'C:\\Program Files (x86)\\VSCodium\\resources\\app\\node_modules.asar\\@vscode\\policy-watcher\\lib\\binding\\node-v110-win3
  phase: 'loading',iles (x86)\VSCodium\resources\app\out\main.js:5:3988
  moduleId: '@vscode/policy-watcher',se [as oncomplete] (node:internal/fs/read_file_context:68:3) {
  neededBy: [ 'vs/platform/policy/node/nativePolicyService' ]
}   'C:\\Program Files (x86)\\VSCodium\\resources\\app\\node_modules.asar\\@vscode\\policy-watcher\\build\\vscode-policy-watcher
Here are the modules that depend on it:
[ 'vs/platform/policy/node/nativePolicyService' ]
~~~~

And then it just hangs.

**Comment by yangu-hury** [comment]
I'm having the same problem with x86 versions.

**Comment by daiyam** [comment]
The `1.83.0` should fix the issue.
Something has changed on how to build with custom node-gyp.
It was one of the error in #1669

**Comment by daiyam** [comment]
`1.83.0.23283` should fix the issue. It will be out in about an hour.

I've tested the `ia32` version on Windows 10 x64 and it was working

**Comment by blachawk** [comment]
Thanks for the new release.  

For anyone who originally installed VSCodium via Winget through Powershell on Windows 10

- I opened up my Powershell and run **winget upgrade**
- I saw the new VSCodium version on the list
- I then ran **winget upgrade VSCodium**
- But the installer stopped early in the process due to issue. While that installation window was open, I opened up my Windows Task Manager, then went to the Details tab, and killed all the vscodium processes running (except for the current installation window running).  
- I then went back to my installation window and was able to continue installing the update.

Now when I run VSCodium, it opens normally again.

What I didn't know what the automatic silent updates going on behind the scenes.  Is there a way to turn that off so we manually update VSCodium when we want?
